zinit-install.zsh

```diff
@@ -1455,8 +1455,8 @@ builtin source "${ZINIT[BIN_DIR]}/zinit-side.zsh" || {
         amd64   "(x86_64|amd64|intel|linux64|linux_amd64)"
         aarch64 "aarch64"
         aarch64-2 "arm"
-        musl "*musl*"
-        darwin  "(darwin|mac|macos|osx|os-x)"
+        musl "*((#s)|/)*(linux|musl)*((#e)|/)*"
```

Comment:
Please add comments, the next person that reads this will be grateful. It might be you :)

Reply:
lmao, good catch